### DISCUSSION TITLE
Fixing duplicates in mysql (disk 0+1)

### DISCUSSION THREAD

**Discussion by rpridemore** [ampache/ampache#3968] [Q&A] [answered]
I am a long-time user of Ampache and have a large digital music collection.  I have db modifications where the metadata in my audio files is not always correct or to my liking.  This has always been fine, but the recent addition of new tables such as album_disk (and whatever has populated that table on an update) seems to have caused issues in the UI.  I normally stream music to a player on my phone using the subsonic api, which seems unaffected.  But on the web UI, I see each album twice:  once with [Disk 0] and again with [Disk 1].  How would I go about fixing this?
I really do not care about disk - this is digital music and I don't care how many songs fit on a CD or whatever!

**Comment by lachlan-00** [comment]
update from tags on the artist/album or verify on the catalog

there must not have been the tags set for you earlier and the updates have created separate items when splitting them into disk objects

**Reply by lachlan-00** [reply]
i think in the older days disk0 was used when not detected and rolled to 1 automatically, now we don't change the value and default to 1 so i think a verify is going to be enough

**Reply by rpridemore** [reply, marked as answer]
I am not a php dev, but know sql.  I think I recovered everything by setting disk to 1 in all album rows.  It is a nullable column, but whatever.  Then set disk_count in album to 1.  Same in album_disk and remove where disk <> 1.

I would simply start over and index from metadata on the files, but I have ratings for most of my collection and use smart-playlists based on those.  So losing all that would be a real pain.

After what I did, it seems I lost the album ratings, but still have them for songs, which is what I mostly care about.

**Comment by jamesc2151** [comment]
I believe what you looking for is in table song and fields album album_disk and disk
